pom.xml

```diff
@@ -65,7 +80,12 @@
 			<artifactId>log4j-core</artifactId>
 			<version>2.19.0</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
use junit 5

Reply:
done

Reply:
I think Spring already comes with Junit 5, no need to declare explicitly

Reply:
As I read, in some versions unit test fails, because of it I specified a non-broken version.